### DISCUSSION TITLE
Добро пожаловать в Torpeda Discussion

### DISCUSSION THREAD

**Discussion by V8ST8K** [nordbearbotdev/Torpeda#2] [Announcements]
## Добро Пожаловать 👋!!!

Мы используем обсуждения как место для общения с другими членами нашего сообщества. Мы надеемся, что вы будете:
 * Задавать вопросы, которые вас интересуют. 
 * Делится идеями. 
 * Общайтся с другими членами сообщества. 
 * Приветствовать других и быть непредубежденными.

Помните, что это сообщество, которое мы строим вместе💪.

Чтобы начать, прокомментируйте ниже, представившись, и расскажите нам о том, что вы делаете с этим сообществом.

**Comment by seveneven7** [comment]
👍

**Comment by V8ST8K** [comment]
> 👍

Приветствую, @seveneven7

**Reply by seveneven7** [reply]
И тебе привет дружище !)

**Reply by V8ST8K** [reply]
> И тебе привет дружище !)

Обновление)))

![IMG-20220205-WA0002(1).jpg](https://user-images.githubusercontent.com/85753549/152675862-8546bf97-5374-4b55-9624-61989a79d261.jpg)

**Reply by seveneven7** [reply]
Вау выглядит очень классно!
Будто из фильмов про хакеров

**Reply by V8ST8K** [reply]
> Вау выглядит очень классно!
> Будто из фильмов про хакеров 

Хехех))). Спасибо.
